conda/shell/condabin/_conda_activate.bat

```diff
@@ -38,10 +38,10 @@
 )
 @ECHO Failed to create temp directory "%TMP%\conda-<RANDOM>\" & exit /b 1
 :tmp_file_created
-@"%CONDA_EXE%" %_CE_M% %_CE_CONDA% shell.cmd.exe %* 1>%UNIQUE%
+@"%CONDA_EXE%" %_CE_M% %_CE_CONDA% shell.cmd.exe %* 1>"%UNIQUE%"
 @IF %ErrorLevel% NEQ 0 @EXIT /B %ErrorLevel%
-@FOR /F %%i IN (%UNIQUE%) DO @SET _TEMP_SCRIPT_PATH=%%i
-@RMDIR /S /Q %UNIQUE_DIR%
+@SET /P _TEMP_SCRIPT_PATH=<"%UNIQUE%"
```

Comment:
If I am understanding this correctly, `<"%A_PATH_VAR%` is the CMD syntax to get input from `stdin`, and since we use `SET /P`, it takes the first (and only) line?

Ref: https://stackoverflow.com/a/40163258/3407590